### DISCUSSION TITLE
How to change the font size of the side bar?

### DISCUSSION THREAD

**Discussion by Sylvia-zz** [squidfunk/mkdocs-material#6927] [Q&A]
Hello, when i upgrade my mkdocs-material to higher version （9.5.x）, i found that the font size of the left sidebar and right sidebar of the page become bigger. Only the font size of the sidebar! How to make the font size the same as before? 
![image](https://github.com/squidfunk/mkdocs-material/assets/163830885/4b0c95d1-4cda-4c41-b9b7-288fe8ccfac1)

**Comment by alexvoss** [comment]
Which version were you coming from? Knowing this would help us narrow down where this might have changed. A [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) would help us debug this on our own system.

**Reply by Sylvia-zz** [reply]
Maybe 9.1 or 9.2.  I just found there was a new version and then upgraded. Sorry about that
